pyftpdlib/handlers.py

```diff
@@ -3145,7 +3145,7 @@ def __init__(self, *args, **kwargs):
             self._ssl_want_write = False
 
         def readable(self):
-            return self._ssl_want_read or super().readable()
+            return self._ssl_accepting or self._ssl_want_read or super().readable()
```

Comment:
As a micro optimization, please do:

```python
return self._ssl_want_read or self._ssl_accepting  or super().readable()
```

`want_read` is supposed to happen much more often than `accepting`